src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.System/Properties/Resources.resx

```diff
@@ -0,0 +1,412 @@
+  </data>
+  <data name="Microsoft_plugin_sys_restart_computer_description" xml:space="preserve">
+    <value>Restart computer</value>
+    <comment>This should align to the action in Windows of a restarting your computer.</comment>
+  </data>
+  <data name="Microsoft_plugin_sys_Search_NetworkKeywordList" xml:space="preserve">
+    <value>ip; mac; address</value>
+    <comment>Translate 'ip' as 'ip' and not as 'ip address'. Same for 'mac'.)</comment>
+  </data>
+  <data name="Microsoft_plugin_sys_shutdown_computer" xml:space="preserve">
+    <value>Shutdown</value>
+    <comment>This should align to the action in Windows of a shutting down your computer.</comment>
+  </data>
+  <data name="Microsoft_plugin_sys_shutdown_computer_confirmation" xml:space="preserve">
+    <value>You are about to shut down this computer, are you sure?</value>
+    <comment>This should align to the action in Windows of a shutting down your computer.</comment>
+  </data>
+  <data name="Microsoft_plugin_sys_shutdown_computer_description" xml:space="preserve">
+    <value>Shutdown computer</value>
+    <comment>This should align to the action in Windows of a shutting down your computer.</comment>
+  </data>
+  <data name="Microsoft_plugin_sys_sign_out" xml:space="preserve">
+    <value>Sign out</value>
+    <comment>This should align to the action in windows of a signing out back to the lock screen.</comment>
+  </data>
+  <data name="Microsoft_plugin_sys_sign_out_confirmation" xml:space="preserve">
+    <value>You are about to sign out of this computer, are you sure?</value>
+    <comment>This should align to the action in windows of a signing out back to the lock screen.</comment>
+  </data>
+  <data name="Microsoft_plugin_sys_sign_out_description" xml:space="preserve">
+    <value>Sign out of computer</value>
+    <comment>This should align to the action in windows of a signing out back to the lock screen.</comment>
+  </data>
+  <data name="Microsoft_plugin_sys_sleep" xml:space="preserve">
+    <value>Sleep</value>
+    <comment>This should align to the action in Windows of a making your computer go to sleep.</comment>
+  </data>
+  <data name="Microsoft_plugin_sys_sleep_confirmation" xml:space="preserve">
+    <value>You are about to put this computer to sleep, are you sure?</value>
+    <comment>This should align to the action in Windows of a making your computer go to sleep.</comment>
+  </data>
+  <data name="Microsoft_plugin_sys_sleep_description" xml:space="preserve">
+    <value>Put computer to sleep</value>
+    <comment>This should align to the action in Windows of a making your computer go to sleep.</comment>
+  </data>
+  <data name="Microsoft_plugin_sys_Speed" xml:space="preserve">
+    <value>Speed</value>
+  </data>
+  <data name="Microsoft_plugin_sys_State" xml:space="preserve">
+    <value>State</value>
+  </data>
+  <data name="Microsoft_plugin_sys_SubTitle_CopyHint" xml:space="preserve">
+    <value>Select to copy</value>
```

Comment:
OKay I'm definitely picking nits here, but I don't think we need to add this text to all the subtitles. I think it ends up redundant with the command name:

![image](https://github.com/user-attachments/assets/81b1ec38-c97d-4492-9f72-b8bc438b1b61)